addons/kibana/kibana.yaml

```diff
@@ -29,14 +29,13 @@ spec:
     - matchLabels:
         kubeaddons.mesosphere.io/name: elasticsearch
   chartReference:
-    chart: stable/kibana
-    version: 3.2.7
+    chart: elastic/kibana
+    repo: https://helm.elastic.co
+    version: 6.8.12
     valuesRemap:
       "ingress.annotations.traefik\\.ingress\\.kubernetes\\.io/auth-url": "ingress.auth.auth-url"
     values: |
       ---
-      image:
-        tag: 6.8.10
```

Comment:
@alejandroEsc @branden I wasn't sure why the image was pinned to 6.8.10 here, but if there is a particular reason why I can revert

Reply:
could you check the default version? I am not sure about this either, odd.

Reply:
The default (for 6.8.12 chart) is, unsurprisingly, 6.8.12 :p same as the previous version 6.8.10